src/main/java/seedu/moneygowhere/data/currency/Currency.java

```diff
@@ -0,0 +1,23 @@
+package seedu.moneygowhere.data.currency;
+
+/**
+ * Stores data associated with a currency.
+ */
+public class Currency {
+    private String currencyCode;
+    private Float rate;
```

Comment:
Do use BigDecimal instead of floats or doubles when representing currency amounts.